### PR TITLE
add code

### PR DESCRIPTION
**Pull request by stevelee2497**

### CODE REVIEW

File: Exercise5/Controllers/Adapters/AssignmentHolder.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                assignment = value;
+                tvCourse.Text = assignment.CourseName + ", ";
+                tvExercise.Text = assignment.Exercise + ", ";
+                tvLesson.Text = assignment.LessonName;
+                tvDeadline.Text = assignment.Deadline.ToShortDateString();
+                tvDuration.Text = assignment.Duration;
+                var timeRemainder = (assignment.Deadline - DateTime.Now).Days;
+                if (timeRemainder >= 0)
+                {
+                    btnStatus.Text = "  " + timeRemainder + (timeRemainder == 1 ? " Day left" : " Days left");
```
Comment (voquanghoa) [inline Exercise5/Controllers/Adapters/AssignmentHolder.cs]: The code is quite duplicated. Please extract a method with some params and reuse.

File: Exercise5/Controllers/DataProvider.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        public static DataProvider Instance => instance ?? (instance = new DataProvider());
+
+        private DataProvider()
+        {
+        }
+
+        public void LoadSampleData()
+        {
+            Assignments = new List<Assignment>
```
Comment (voquanghoa) [inline Exercise5/Controllers/DataProvider.cs]: Move this code into the contructor.

File: Exercise5/Controllers/Adapters/AssignmentHolder.cs (hunk 1 above)
Comment (stevelee2497) [reply Exercise5/Controllers/Adapters/AssignmentHolder.cs]: done

File: Exercise5/Controllers/DataProvider.cs (hunk 2 above)
Comment (stevelee2497) [reply Exercise5/Controllers/DataProvider.cs]: done

**Pull request merged**